integration-tests/container/perf_event_open/sched_process_exit.c

```diff
@@ -0,0 +1,62 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+
+#include <asm/unistd.h>
+#include <linux/perf_event.h>
+#include <sys/ioctl.h>
+
+static long
+perf_event_open(struct perf_event_attr* hw_event, pid_t pid,
+                int cpu, int group_fd, unsigned long flags) {
+  int ret;
+
+  ret = syscall(__NR_perf_event_open, hw_event, pid, cpu,
+                group_fd, flags);
+  return ret;
+}
+
+int main(int argc, char** argv) {
+  struct perf_event_attr pe;
```

Comment:
I see `pe` is initialized with a memset on line 36, we can probably omit that and initialize it here instead.
```suggestion
  struct perf_event_attr pe = {0};
```

Reply:
We could also initialize most of its fields, but I leave that up to you if you want to.